lib/mappers/bhl_mods.py

```diff
@@ -0,0 +1,210 @@
+from akara import logger
+from dplaingestion.utilities import iterify
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.oai_mods_mapper import OAIMODSMapper
+
+
+class BHLMapper(OAIMODSMapper):
+    def __init__(self, provider_data):
+        super(BHLMapper, self).__init__(provider_data)
+
+    def map_date_and_publisher(self):
+        prop = self.root_key + "originInfo"
+
+        _dict = {
+            "date": None,
+            "publisher": []
+        }
+
+        if exists(self.provider_data, prop):
+            for s in iterify(getprop(self.provider_data, prop)):
+                if "dateOther" in s:
+                    date_list = iterify(s.get("dateOther"))
+                    date = [t.get("#text") for t in date_list if
+                            t.get("keyDate") == "yes"
+                            and t.get("type") == "issueDate"]
+                    # Check if last date is already a range
+                    if "-" in date[-1] or "/" in date[-1]:
+                        _dict["date"] = date[-1]
+                    elif len(date) > 1:
+                        _dict["date"] = "%s-%s" % (date[0], date[-1])
+                    else:
+                        _dict["date"] = date[0]
+
+                if "publisher" in s:
+                    _dict["publisher"].append(s.get("publisher"))
+
+            self.update_source_resource(self.clean_dict(_dict))
+
+    def map_description(self):
+        prop = self.root_key + "note"
+
+        if exists(self.provider_data, prop):
+            description = []
+            for s in iterify(getprop(self.provider_data, prop)):
+                try:
+                    if s.get("type") == "content":
+                        description.append(s.get("#text"))
+                except:
+                    logger.error("Error getting note/type from record %s" %
+                                 self.provider_data["_id"])
+
+            self.update_source_resource({"description": description})
+
+    # OAI MODS Mapper is blowing up on this method because of arg c
+    # ount issues, even though we handle it below
```

Comment:
Not sure what the issue could be, but there are some other MODS mappers with various strategies.  `missouri_mapper.py` has some complex cases, whereas `harvard_mapper.py` and `bpl_mapper.py` were successful extending `OAIMODSMapper`'s `map_format()`.

Meanwhile, `digitalnc_mapper.py` takes the same approach of passing. I can't find anything in issues.dp.la indicating why this mapper was modified in https://github.com/dpla/ingestion/commit/b022a81ec2d to pass on `map_format()`, so it's possible that's not relevant; though it may be a clue.